libraries/GCS_MAVLink/include/mavlink/v1.0/ardupilotmega/version.h

```diff
@@ -5,7 +5,7 @@
 #ifndef MAVLINK_VERSION_H
 #define MAVLINK_VERSION_H
 
-#define MAVLINK_BUILD_DATE "Wed Jul 20 13:14:21 2016"
+#define MAVLINK_BUILD_DATE "Thu Jul 21 14:44:45 2016"
```

Comment:
I don't know. I think that work is only theoretical at the moment. It isn't compatible with the changes on @priseborough 's PR branch, but with the rebase protocol for PRs I'm not sure if it's better to try to reconcile them both before merging or just merge one and force a rebase of the other to address the differences. With Ben's changes to the generator that would touch a pretty small set of files.